src/antelope/stores/evm.ts

```diff
@@ -232,6 +264,15 @@ export const useEVMStore = defineStore(store_name, {
                 throw new AntelopeError('antelope.evm.error_no_provider');
             }
         },
+        // utils ---
+        toWei(value: string | number, decimals: number): string {
+            const amount = typeof value === 'string' ? parseFloat(value) : value;
+            const amountInWei = (amount * Math.pow(10, decimals)).toString();
```

Comment:
this will break, arithmetic should be done using bignumber library like BigNumber(whatever).pow(10)